pmm-app/src/pmm-add-instance/components/Discovery/Discovery.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import Discovery from './Discovery';
+
+jest.mock('shared/components/helpers/notification-manager');
+
+describe('Discovery instance:: ', () => {
+  it('Should render correct', () => {
+    const selectInstance = jest.fn();
+
+    const root = mount(<Discovery selectInstance={selectInstance} />);
+
+    expect(root.find('#input-aws_access_key-id').length).toBe(1);
```

Comment:
Should we use html `id` attributes, knowing that they must be unique? Is there a way to guarantee it?

Reply:
Agree, I used it because didn't notice that field generates data-qa attribute too, replaced